plugin/trino-hive/src/main/java/io/trino/plugin/hive/security/AccessControlMetadata.java

```diff
@@ -120,6 +120,14 @@ default void grantSchemaPrivileges(ConnectorSession session, String schemaName,
         throw new TrinoException(NOT_SUPPORTED, "This connector does not support grants on schemas");
     }
 
+    /**
+     * Get the owner on the specified schema
+     */
+    default Optional<HivePrincipal> getSchemaOwner(ConnectorSession session, String schemaName)
+    {
+        return Optional.empty();
```

Comment:
Should this throw in the default implementation?
Looks like all other methods do throw by default.